src/edit/Toolbar/PosterFrameDialog/CustomPosterUpload.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="text-left">
+    <p class="my-3">or upload an image from your computer</p>
+    <upload-btn
+      @change="$emit('upload', $event)"
+      label="Upload image"
+      accept="image/jpeg"
+      small depressed>
+      <template #icon>
+        <v-icon>mdi-upload</v-icon>
+      </template>
+    </upload-btn>
+    <p :class="{ 'error--text': isError }" class="mt-1 text-caption">
+      {{ fileSizeMessage }}
+    </p>
```

Comment:
Why not generalize this as an `errorMessage`?
```suggestion
    <p :class="{ 'error--text': errorMessage }" class="mt-1 text-caption">
      {{ errorMessage }}
    </p>
```